Branch-SDK/src/io/branch/referral/Branch.java

```diff
@@ -291,8 +291,11 @@
 
     private static boolean isLogging_ = false;
 
-    private static boolean checkInstallReferrer_ = false;
-    private static long PLAYSTORE_REFERRAL_FETCH_WAIT_FOR = 5000;
+    // Branch will wait for 1.5 sec to capture the install referrer inorder to provide
+    // accurate attribution . This will be only on the first time user open the app.
+    // @see to skip this please call {@link #disablePlayStoreReferrer() }
+    private static boolean checkInstallReferrer_ = true;
```

Comment:
Yeah I saw it's only for installs, that's when you want the user to have the least delayed experience possible IMO. In my metric lacking opinion, playstore referrer information is a less utilized/more advanced feature(?). If you think **every** dev can benefit from this feature then it should be on, but our on-boarding/integration docs will have to cover it, I can already see the tickets in support "why does my app take longer to boot up on install???? I only want deep linking, why does it have to wait for something I'M NOT USING????!?!"